apps/calendar/js/backend/calendar_service.js

```diff
@@ -1,88 +1,82 @@
+define(function(require, exports, module) {
 'use strict';
-importScripts('../ext/alameda.js');
 
-require.config({
-  baseUrl: '/js/backend',
-  paths: {
-    ext: '/js/ext'
-  }
+var Db = require('db');
+var threads = require('ext/threads');
+
+var db = new Db('b2g-calendar');
+var service = threads.service('calendar');
+
+service.method('start', () => db.load());
```

Comment:
what if we avoid this method and call the `db.load` automatically when trying to access any data? since the "stores" will really be in the worker I think that it might make more sense and avoid headaches in the long run (specially if we end up having multiple clients)